Bootstrap.php

```diff
@@ -442,6 +452,14 @@ protected function registerMyEvents()
         );
     }
 
+    /**
+     * Registers dependencies / autoloader
+     */
+    public function registerMyComponents()
+    {
+        require_once $this->Path() . '/vendor/autoload.php';
```

Comment:
Where is the `.env` file loaded?

Reply:
It comes from the SDK, ```vendor/nosto/php-sdk/src/.env```, the autoloader takes care of loading the vendor assets, and so on..

Reply:
👍